src/CoreRegistry.cpp

```diff
@@ -12,22 +12,33 @@
 
 namespace visor {
 
-CoreRegistry::CoreRegistry(HttpServer *svr)
-    : _svr(svr)
+CoreRegistry::CoreRegistry()
 {
 
     _logger = spdlog::get("visor");
     if (!_logger) {
         _logger = spdlog::stderr_color_mt("visor");
     }
 
+    // inputs
+    _input_manager = std::make_unique<InputStreamManager>();
```

Comment:
Only a question: is this module dynamically loaded? If so, I didn't get how it is created before it os loaded @weyrick

Reply:
no, this one is static and part of core code

Reply:
Oh ok, nice